### ISSUE TITLE
Select _PARTITION_COLUMN while using Spark BQ Connector

### ISSUE THREAD

**Issue by debpriyaseal** [GoogleCloudDataproc/spark-bigquery-connector#364]
Hi 
I am trying to read BQ dataset using PySpark. And would need the `_PARTITIONTIME` as column in the output dataframe. 
As I am reading multiple dates of data. And need to know which row belongs to which date. 
I am trying below command:
```
val df = spark.read.format("bigquery")
  .option("filter", "_PARTITIONDATE > '2019-01-01'")
  ...
  .load(TABLE)
```
However, I am not seeing the `_PARTITIONTIME` in output. Can someone tell me how to do this? I know running BQ SQL from Spark is one option. Is there any option I can add above? Couldn't find anything [here](https://github.com/GoogleCloudDataproc/spark-bigquery-connector#properties)

**Comment by davidrabinowitz** [comment]
Will be solved once there's an API support for it and PR #286 is merged to master

**Comment by davidrabinowitz** [comment]
Fixed in version 0.20.0
